rakelib/persistent_attachment_remediation.rake

```diff
@@ -35,7 +35,7 @@ def update_ipf_form_data(attachment, claim, ipf_form_data, key, destroyed_names)
       (att.respond_to?(:confirmationCode) && att.confirmationCode == attachment.guid)
   end
 
-  puts "Add destroyed attachment file to list: #{mask_file_name(destroyed_attachment&.name)}"
+  puts "Add destroyed attachment file to list: #{destroyed_attachment&.name}"
```

Comment:
Why is `mask_file_name` getting removed?

Reply:
We want to take note of the file names in case we can delete them later. It's only during the output of the rake task and not leaked anywhere else.